components/eam/bld/namelist_files/use_cases/2010_eam_CMIP6_chemUCI-Linoz-mam5-vbs.xml

```diff
@@ -69,10 +69,10 @@
 
 <!-- <tracer_cnst_filelist>this_field_is_not_used</tracer_cnst_filelist> -->
 <tracer_cnst_type    >CYCLICAL</tracer_cnst_type>
-<tracer_cnst_cycle_yr>1995</tracer_cnst_cycle_yr>
-<tracer_cnst_file    >ch4_oxid_1.9x2.5_L26_1990-1999clim.c090804.nc</tracer_cnst_file>
+<tracer_cnst_cycle_yr>2015</tracer_cnst_cycle_yr>
```

Comment:
2010 instead of 2015?

Reply:
The tracer cnst file proposed to use only has 2005 and 2015. I use 2015 following v1 and v2 practice.